### PR TITLE
Skip exit in wp_die() function

### PR DESCRIPTION
**Pull request by wppunk**
Fixes: #28 

I've skipped `die` in the `wp_die` function.

### CODE REVIEW

File: src/WpDie.php [hunk 1]
```diff
@@ -16,7 +16,7 @@ private function __construct() {
 
 	public function die_handler( $function ) {
 
-		return [ $this, $function ];
+		return array( $this, $function );
```
Comment (szepeviktor) [inline src/WpDie.php:19]: @wppunk You could replace all other `array()` -s :)

Comment (wppunk) [reply src/WpDie.php:19]: @szepeviktor What do you mean? Other arrays have a long syntax

Comment (szepeviktor) [reply src/WpDie.php:19]: e.g. `$args = array()`

File: src/WpDie.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			'wp_die_json_handler'   => '_json_wp_die_handler',
+			'wp_die_jsonp_handler'  => '_json_wp_die_handler',
+			'wp_die_xmlrpc_handler' => '_xmlrpc_wp_die_handler',
+			'wp_die_xml_handler'    => '_xml_wp_die_handler',
+		];
+		add_filter( 'wp_die_ajax_handler', [ $this, 'change_handler' ], 10, 1 );
+		add_filter( 'wp_die_json_handler', [ $this, 'change_handler' ], 10, 1 );
+		add_filter( 'wp_die_jsonp_handler', [ $this, 'change_handler' ], 10, 1 );
+		add_filter( 'wp_die_xmlrpc_handler', [ $this, 'change_handler' ], 10, 1 );
+		add_filter( 'wp_die_xml_handler', [ $this, 'change_handler' ], 10, 1 );
```
Comment (szepeviktor) [inline src/WpDie.php:27]: @wppunk Would you `foreach( array_keys( $this->core_handlers ) ) ... ` ?

**Review by szepeviktor — CHANGES_REQUESTED** [review]

**Comment by szepeviktor** [comment]
> Short array syntax is not allowed

This is so hard!

**Comment by leogermani** [comment]
Thanks for the PR!

A question worth asking. Do we want these filters enabled by default in all cases? Are there any situations when we would not want to activate this?

**Comment by szepeviktor** [comment]
> Do we want these filters enabled by default in all cases?

Yes, it is reasonable to make it options. Although I do not know how.

**Comment by wppunk** [comment]
@leogermani I think these filters are important by default because use the `die` function in the test stops it. This PR only starts for this and I sure next you can add other cases when need to prevent the `die` function.

**Review by leogermani — APPROVED** [review]
Let's merge it for now.

I think we can think of a way to disable this and other modules if we want, maybe `Wordbless\load()` could get a parameter to define that

**Comment by szepeviktor** [comment]
That seem reasonable.
But could think of a test case that want core to execute `exit;` ?

**Pull request merged**